include/swift/SIL/AbstractionPattern.h

```diff
@@ -179,15 +179,24 @@ class AbstractionPattern {
     /// type.  ObjCMethod is valid.  OtherData is an encoded foreign
     /// error index.
     ObjCMethodType,
-    /// The uncurried imported type of a C++ method. OrigType is valid and is a
-    /// function type. CXXMethod is valid.
+    /// The uncurried imported type of a C++ non-operator method. OrigType is
```

Comment:
While you're rewriting these comments, could you replace "method" with "non-static member function" where this term is referring to C++?